nix/cardano-services/deployments/stake-pool-provider-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: stake-pool-provider
+    network: preview
+    release: dev-preview-cardanojs
+  name: dev-preview-cardanojs-stake-pool-provider
+spec:
+  selector:
+    matchLabels:
+      app: stake-pool-provider
+      network: preview
+      release: dev-preview-cardanojs
+  template:
+    metadata:
+      labels:
+        app: stake-pool-provider
+        network: preview
+        release: dev-preview-cardanojs
+    spec:
+      containers:
+        - args:
+            - start-provider-server
+          env:
+            - name: ALLOWED_ORIGINS
+              value: chrome-extension://gafhhkghbfjjkeiendhlofajokpaflmk,chrome-extension://efeiemlfnahiidnjglmehaihacglceia
+            - name: DISABLE_STAKE_POOL_METRIC_APY
+              value: "true"
+            - name: ENABLE_METRICS
+              value: "true"
+            - name: LOGGER_MIN_SEVERITY
+              value: debug
+            - name: NETWORK
+              value: preview
+            - name: OGMIOS_SRV_SERVICE_NAME
+              value: dev-preview-cardano-stack.dev-preview.svc.cluster.local
+            - name: PAGINATION_PAGE_SIZE_LIMIT
+              value: "5500"
+            - name: POSTGRES_DB_STAKE_POOL
+              value: stakepool
+            - name: POSTGRES_HOST_STAKE_POOL
+              value: dev-preview-dbsync-db
+            - name: POSTGRES_PASSWORD_STAKE_POOL
+              valueFrom:
+                secretKeyRef:
+                  key: password
+                  name: stakepool-owner-user.dev-preview-dbsync-db.credentials.postgresql.acid.zalan.do
+            - name: POSTGRES_POOL_MAX_STAKE_POOL
+              value: "10"
+            - name: POSTGRES_PORT_STAKE_POOL
+              value: "5432"
+            - name: POSTGRES_SSL_CA_FILE_STAKE_POOL
+              value: /tls/ca.crt
+            - name: POSTGRES_SSL_STAKE_POOL
+              value: "true"
+            - name: POSTGRES_USER_STAKE_POOL
+              valueFrom:
+                secretKeyRef:
+                  key: username
+                  name: stakepool-owner-user.dev-preview-dbsync-db.credentials.postgresql.acid.zalan.do
+            - name: SERVICE_NAMES
+              value: stake-pool
+            - name: TOKEN_METADATA_SERVER_URL
+              value: http://dev-preview-cardano-stack-metadata.dev-preview.svc.cluster.local
+            - name: USE_TYPEORM_STAKE_POOL_PROVIDER
+              value: "true"
+          image: 926093910549.dkr.ecr.us-east-1.amazonaws.com/cardano-services-server:aana7a352vdaz0rs41yhfh3s85inw004
```

Comment:
:roll_eyes:

Reply:
This thing exists everywhere. Should be removed

Reply:
Originally I was hoping that `ghcr.io` would have worked and we would have been able to publish a `latest` tag, here. But alas!

Reply:
I've backlogged the original intent in [LW-8024](https://input-output.atlassian.net/browse/LW-8024)

Reply:
This would be my [working suggestion](https://github.com/input-output-hk/cardano-js-sdk/pull/864/commits/25beff27b46b1b81ee01b0f1bc2a9fa9991cd8e8) for now.

Reply:
Yeah, it's not good. The only other alternative I could come up with would be to set a fake name `UPDATE-ME` but we also don't need it to make much more complicated than it is and could defer to LW-8024, which is the desired proper solution either way.

Would that be ok? I reverted https://github.com/input-output-hk/cardano-js-sdk/pull/864/commits/25beff27b46b1b81ee01b0f1bc2a9fa9991cd8e8

Reply:
Just delete the line? At least that is what I'd do

I am okay with some gibberish like `UPDATE-ME`  since that would also prevent it from failing silently

Reply:
Implemented in https://github.com/input-output-hk/cardano-js-sdk/pull/864/commits/80e4eae962d7b5d7f01bfd29aff952a87cff8e76